content/en/actions/app_builder/saved_actions.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Save and Reuse Actions"
+description: Save and reuse an action and its parameters
+disable_toc: false
+disable_sidebar: false
+algolia:
+  tags: ['app builder', 'actions']
+type: documentation
+aliases:
+  - /app_builder/actions_catalog/action_blueprints/
+  - /service_management/app_builder/actions_catalog/saved_actions/
+  - /service_management/app_builder/actions/saved_actions/
+  - /service_management/app_builder/saved_actions
+further_reading:
+- link: "/actions/workflows/actions/"
+  tag: "Documentation"
+  text: "Learn about actions in workflows"
+- link: "/integrations/"
+  tag: "Documentation"
+  text: "Learn about integrations"
+
+---
+
+Use the _Saved Actions_ feature to store and reuse an action and its parameters. You can insert a saved action into your app as a new step, or use one to populate an existing step's parameters.
+
+## Save an action
+
+1. From [your apps list][1], hover over an app and click the **Edit** {{< img src="icons/pencil.png" inline="true" style="width:14px;">}} icon.
+1. Click the **Actions** {{< img src="icons/kebab.png" inline="true" style="width:14px;">}} menu, then **Save Action**.
```

Comment:
Can you give some more context on where this button is? It can be easy to get lost in all the panes on these pages!

I also was confused about why you would go in to edit something, then save it without making changes. Is it like duplicating an existing action?